lib/interactor/contract/contract.rb

```diff
@@ -0,0 +1,171 @@
+module Interactor
+  module Contract
+    VALID_TYPES = [:open, :closed]
```

Comment:
Since this PR already fails on 1.9.3 (and there was a discussion of dropping support, since it's no longer supported) we could use `%i(open closed)` instead. Totally a personal perference though!